modules/nw-multus-configure-dualstack-ip-address.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_content-type: PROCEDURE 
+
+[id="nw-multus-configure-dualstack-ip-address_{context}"]
+= Creating a configuration for assignment of dual-stack IP addresses dynamically
+
+Dual-stack IP address assignment can be configured with the `ipRanges` parameter for:
+* IPv4 addresses
+* IPv6 addresses
+* multiple IP address assignment 
+
+.Procedure
+
+. Set type to whereabouts.
+. Use `ipRanges` to allocate IP addresses.
+
+[source,yaml]
+----
+cniVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  additionalNetworks:
+  - name: whereabouts-shim
+    namespace: default
+    type: Raw
+    rawCNIConfig: |-
+      {
+       "name": "whereabouts-dual-stack",
+       "cniVersion": "0.3.1,
+       "type": "bridge",
+       "ipam": {
+         "type": "whereabouts",
+         "ipRanges": [
+                  {"range": "192.168.10.0/24"},
+                  {"range": "2001:db8::/64"}
+              ]
+       }
+      }
+
+
+----
+
+. Attach network to a pod. For more information, see "Adding a pod to an additional network".
+
+. Verify that all IP addresses are assigned.
+
+Run the following command to ensure the IP adresses are assigned as meta-data.
```

Comment:
```suggestion
. Run the following command to ensure the IP addresses are assigned as metadata.
```

Reply:
I think this last one should be a step? And then indent the commands below using `+`? I'm kind of confused because the preview shows "run the following two commands" - maybe it just hasn't updated yet. But there is a typo in `addresses` and `metadata`.

Reply:
Should only say run the following command